File: Android UI/app/src/main/java/io/github/froger/instamaterial/models/StBusiModel.java
package io.github.froger.instamaterial.models;

import java.io.Serializable;

/**
 * Created by dell on 2016/12/14.
 */

public class StBusiModel implements Serializable
{
    public String story_id;
    public String user_id;
    public String nickname;
    public String title;
    public String content;
    public int like_num;
    public boolean isLike;
}

File: WebService/WebService1/Model/StBusiModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace STWebService.Model
{
    public class StBusiModel
    {
        public String story_id { get; set; }
        public String user_id { get; set; }
        public String title { get; set; }
        public String nickName { get; set; }
        public String avatar { get; set; }
        public String photo { get; set; }
        public String content { get; set; }
        public String like_num { get; set; }
        public String bl { get; set; }
    }
}